src/hub.html

```diff
@@ -162,6 +162,34 @@
                 </a-entity>
             </template>
 
+            <template id="static-media">
+                <a-entity
+                    class="interactable"
+                    body="type: static; shape: none; mass: 1;"
+                    auto-scale-cannon-physics-body
+                    set-sounds-invisible
+                >
+                </a-entity>
+            </template>
+
+            <template id="static-controlled-media">
```

Comment:
Confused as to how this actually ends up being not movable?

Reply:
I made the body static and the network schema does not sync transform properties.